src/icon/smokeIcon.tsx

```diff
@@ -37,93 +38,51 @@ class SmokeIcon extends PureComponent<SmokeIconProps> {
-            />
-            <polyline
-              stroke={color.iconFaded}
-              strokeLinecap="round"
-              // tslint:disable-next-line:max-line-length
-              transform="translate(1.603011, 9.642054) rotate(-150.000000) translate(-1.603011, -9.642054)"
-              points="1.10301136 7.64205363 1.10301136 11.6420537 1.10301136 11.6420537"
-            />
-            <polyline
-              stroke={color.iconFaded}
-              strokeLinecap="round"
-              // tslint:disable-next-line:max-line-length
-              transform="translate(3.933013, 10.982051) rotate(-150.000000) translate(-3.933013, -10.982051)"
-              points="3.43301316 8.98205132 3.43301316 12.9820513 3.43301316 12.9820513"
-            />
-            <path
-              stroke={color.iconFaded}
-              strokeLinecap="round"
-              // tslint:disable-next-line:max-line-length
-              transform="translate(10.330288, 6.760456) scale(-1, 1) rotate(-30.000000) translate(-10.330288, -6.760456)"
-              // tslint:disable-next-line:max-line-length
-              d="M7.39847763,5.26702579 L7.39847763,5.26702579 L11.2620983,5.26702579 L11.2620983,5.26702579 C12.3666678,5.26702579 13.2620983,6.1624563 13.2620983,7.26702579 L13.2620983,8.25388607"
-            />
-            <path
-              stroke={color.iconFaded}
-              strokeLinecap="round"
-              // tslint:disable-next-line:max-line-length
-              transform="translate(13.354646, 4.647114) scale(-1, 1) rotate(-30.000000) translate(-13.354646, -4.647114)"
-              // tslint:disable-next-line:max-line-length
-              d="M6.16506325,-0.609455613 L6.16506325,-0.141216059 L6.16506325,-0.141216059 C6.16506325,2.06792294 7.95592425,3.85878394 10.1650632,3.85878394 L16.4804851,3.85878394 L16.4804851,3.85878394 C18.642492,3.85878394 20.4132271,5.57672046 20.4786531,7.73773719 L20.5442283,9.90368411"
-            />
-            <path
-              d="M0.5,23.5 L22.5,1.5"
-              stroke={color.danger}
-              fill={color.iconFaded}
-              fillRule="nonzero"
-              strokeLinecap="round"
-              strokeLinejoin="round"
-            />
-            <path
-              d="M0.5,22.5 L22.5,0.5"
-              stroke={color.white}
-              fill={color.iconFaded}
-              fillRule="nonzero"
-              strokeLinecap="round"
-              strokeLinejoin="round"
-            />
-          </g>
-        )}
+        <g>
+          <path
+            fill={finalColor}
```

Comment:
Can't the fill be defined in the `<g>`? That'll make us repeat ourselves a bit less

Reply:
Seems to work 👍